App/Models/Task.cs

```diff
@@ -1,42 +1,43 @@
 namespace App.Models
 {
-public class Task
-{
-    public long Id { get; set; }
-    public string Title { get; set; }
-    public string Description { get; set; }
-    public DateTimeOffset Created { get; set; }
-    public DateTimeOffset Due { get; set; }
-    public bool IsDone { get; set; }
-
-    /// <summary>
-    /// Use -1 for Id if it is not known
-    /// </summary>
-    public Task(long id, string title, string description, DateTimeOffset created, DateTimeOffset due, bool isDone)
+    public class Task
     {
-        Id = id;
-        Title = title;
-        Description = description;
-        Created = created;
-        Due = due;
-        IsDone = isDone;
-    }
+        public static readonly long DEFAULT_ID = -1;
```

Comment:
I saw that DEFAULT_ID is declared as readonly, I think it's better to have it be const (because its value is known at compile time).

Reply:
Ah yes...
We tried to use `const`, but were to lazy to read about it, so ended up just using `static readonly` for the sake of saving time.

Googled it, `const` keyword exists and is exactly what we need in this use case, you are welcome to change it in further commits.